atomicapp/utils.py

```diff
@@ -360,13 +360,17 @@ def getUniqueUUID():
         return data
 
     @staticmethod
-    def loadAnswers(answers_file):
+    def loadAnswers(answers_file, format=None):
         if not os.path.isfile(answers_file):
             raise AtomicAppUtilsException(
                 "Provided answers file does not exist: %s" % answers_file)
 
         logger.debug("Loading answers from file: %s", answers_file)
-        return anymarkup.parse_file(answers_file)
+        try:
+            result = anymarkup.parse_file(answers_file, format=format)
+        except anymarkup.AnyMarkupError:
+            result = anymarkup.parse_file(answers_file)
+        return result
```

Comment:
a couple of things:

- this would be good to add in as a comment because no one else would figure that out
- the fact that anymarkup does not detect an INI file if it has `:` - is that a bug in anymarkup or do INI files really not allow colons in the section headings. If that violates the INI file spec I'd be less happy about using it.